tests/test_reposInButler.py

```diff
@@ -98,15 +99,12 @@ def map_str(self, dataId, write):
 
 class ReposInButler(unittest.TestCase):
 
-    def clean(self):
-        if os.path.exists(os.path.join(ROOT, 'repoOfRepos')):
-            shutil.rmtree(os.path.join(ROOT, 'repoOfRepos'))
-
-    def setup(self):
-        self.clean()
+    def setUp(self):
+        self.testDir = tempfile.mkdtemp(dir=ROOT, prefix='repoOfRepos-')
```

Comment:
not your bug but while your here you could fix "repoOfRepos" to accurately show the test class name "reposInButler"